pyspecdata/core.py

```diff
@@ -38,7 +38,12 @@
 import sympy as sp
 from numpy import r_, c_, nan, inf, pi
 from mpl_toolkits.mplot3d import axes3d
-import textwrap, atexit, scipy, warnings, inspect, re
+import textwrap
+import warnings
+import scipy
+import atexit
+import inspect
+import re
```

Comment:
to pass flake

Reply:
please check your myflake -- it should be
```
#!/usr/bin/bash
flake8 --inline-quotes='"' --max-line-length=79 --ignore=E203,E225,E226,E231,E261,E265,E293,E302,E305,E306,E401,E501,E731,E741,Q002,W291,W293,W503 $*
```